src/lib/components/forms/checkbox/terra-checkbox.component.html

```diff
@@ -3,11 +3,11 @@
      [placement]="tooltipPlacement">
     <label class="input-control checkbox small-check" [ngClass]="{'disabled-chk': inputIsDisabled}">
         <input type="checkbox"
-               [(ngModel)]="innerValue"
+               [(ngModel)]="value"
```

Comment:
Da muss innerValue bleiben, sonst wird die funktionalität verändert
```suggestion
               [(ngModel)]="_innerValue"
```